lib/pantry/helpers/category_helper.rb

```diff
@@ -56,14 +57,15 @@ def determine_large_category(item_name)
 
         def helper_for(large_category)
           {
-            :dairy          => Pantry::Helpers::DairyCategoryHelper
-            # :meat           => Pantry::Helpers::MeatCategoryHelper,
-            # :poultry        => Pantry::Helpers::PoultryCategoryHelper,
-            # :fruit          => Pantry::Helpers::FruitCategoryHelper,
-            # :vegetable      => Pantry::Helpers::VegetableCategoryHelper,
-            # :grain          => Pantry::Helpers::GrainCategoryHelper,
-            # :packaged       => Pantry::Helpers::PackagedCategoryHelper,
-            # :other          => Pantry::Helpers::OtherCategoryHelper
+            :dairy          => Pantry::Helpers::DairyCategoryHelper,
+            :meat           => Pantry::Helpers::MeatCategoryHelper,
+            :poultry        => Pantry::Helpers::PoultryCategoryHelper,
+            :eggs           => Pantry::Helpers::EggCategoryHelper,
+            :seafood        => Pantry::Helpers::SeafoodCategoryHelper,
+            :fruit          => Pantry::Helpers::FruitCategoryHelper,
+            :vegetable      => Pantry::Helpers::VegetableCategoryHelper,
+            :grain          => Pantry::Helpers::GrainCategoryHelper,
+            :other          => Pantry::Helpers::OtherCategoryHelper
           }[large_category]
```

Comment:
completely up to you, but you could probably pull this off with some fhancy meta programming:
```ruby
def helper_for(category)
  "Pantry::Helpers#{category.classify}CategoryHelper".constantize
end
```
benefit would be if you added new categories you wouldn't have to mess with this method. downside is that it's a little cryptic, and it looks like `:eggs` would have to be handled individually or renamed.